src/instructlab/utils.py

```diff
@@ -541,3 +541,20 @@ def split_hostport(hostport: str) -> tuple[str, int]:
     if not hostname or not port:
         raise ValueError(f"Invalid host-port string: '{hostport}'")
     return hostname, port
+
+
+def is_oci_repo(repo_url: str) -> bool:
+    """
+    Checks if a provided repository follows the OCI registry URL syntax
+    """
+
+    # TO DO: flesh this out and make it a more robust check
+    oci_url_prefix = "docker://"
+    return repo_url.startswith(oci_url_prefix)
+
+
+def is_tool_present(name: str) -> bool:
+    # Standard
+    from shutil import which
+
+    return which(name) is not None
```

Comment:
I wonder how this will behave in CI and on systems with different permissions. Will it always pick up the proper bin?

Reply:
The assumption is that if `skopeo` is present it would be on the system path